pyccel/errors/messages.py

```diff
@@ -123,6 +123,10 @@
 INVALID_FILE_EXTENSION = 'Wrong file extension. Expecting `py` of `pyh`, but found'
 INVALID_PYTHON_SYNTAX = 'Python syntax error'
 
+# Assignment errors
+REASSIGN_TARGET = 'can not reassign a target variable'
+ASSIGN_ALLOCATABLES = 'can not assign two allocatable variables'
```

Comment:
I do not understand this message. I cannot do:
```
a = np.ones(3) # assign one allocatable variable
b = np.ones(3) # assign another allocatable variable (now I have assigned two allocatable variables)
```

Reply:
i meant to say: assigning between to alloctables variables
is it the right way to express it?

Reply:
can I merge the two errors to :
"array reallocation will cause unaccessible memory"